utils/hyperlink.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Seems like a good candidate for unit testing various behaviors.

Reply:
I thought the same while coding.. xD